script.js

```diff
@@ -0,0 +1,136 @@
+// Start here
+
+// Step 1 - Welcome and introduction
+// Your code goes here
+alert(
+  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+);
+// Asking for name
+
+  let userName = prompt("What is your name?");
+
+// follow up greeting with name
+
+if (userName) {
+  alert(`Hello ${userName}! Welcome!`);
+} else {
+  alert(`Hello there! Welcome `);
+}
+
+// Step 2 - Food choice
+
+//asking for userChoice
+let chosenPizza;
+
+while (true) {
+
+let userChoice = prompt(`Enter the number of your chosen Pizza: 
+1. Margerita 
+2. Pepperoni 
+3. Hawaiian`);
+
+if (userChoice === "1") {
+  chosenPizza = "Margerita";
+  break;
+} else if (userChoice === "2") {
+  chosenPizza = "Pepperoni";
+  break;
+} else if (userChoice === "3") {
+  chosenPizza = "Hawaiian";
+  break;
+} else {
+  alert("Invalid choise. Please select 1,2 or 3");
+}
+
+}
+alert(`You chose a ${chosenPizza} pizza`);
+
+// Step 3 - Subtype choice
+// Your code goes here
+
+let chosenSize;
+
+while (true) {
+
+let userChoice2 = prompt(`Please select size: 
+1. Small 
+2. Normal
+3. Family-size `);
+
+if (userChoice2 === "1") {
+  chosenSize = "Small";
+  break;
+} else if (userChoice2 === "2") {
+  chosenSize = "Normal";
+  break;
+} else if (userChoice2 === "3") {
+  chosenSize = "Family-size";
+  break;
+} else {
+  alert("Please select a size");
+}
+
+}
+alert(`Thank you for choosing the ${chosenSize} ${chosenPizza} pizza`);
```

Comment:
just an English comment (sorry, I am a teacher and can't help it!) I would change the article 'the' to 'a'